tests/MongoDB.Driver.Tests/Specifications/transactions/TransactionTestRunner.cs

```diff
@@ -399,7 +399,7 @@ private DisposableBundle ConfigureFailPointOnPrimaryOrShardRoutersIfNeeded(IMong
 
         private void ExecuteOperations(IMongoClient client, Dictionary<string, object> objectMap, BsonDocument test)
         {
-            var factory = new JsonDrivenTestFactory(client, _databaseName, _collectionName, bucketName: null, objectMap);
+            var factory = new JsonDrivenTestFactory(this, client, _databaseName, _collectionName, bucketName: null, objectMap);
```

Comment:
This is what happens when you leave in the old constructor for backward compatibility... :)

Was this a merge conflict resolution problem? Looks like it was.

Reply:
Agreed, it probably would have been better to update the old constructor! 

Also agreed: I think it was probably a merge conflict resolution problem